Docs/userguide/on_target_inference.rst

```diff
@@ -27,140 +27,145 @@ to generate both the items. The exported model type will differ based on the fra
    * - TensorFlow
      - .h5 or .pb
 
+You can use Qualcomm\ |reg| AI hub to compile a model and submit an inference job, or use the Qualcomm\ |reg| AI Engine Direct SDK to quantize, compile, and run your model.
 
 Qualcomm\ |reg| AI hub
 ======================
 
-|qai_hub|_ simplifies the AI model deployment on a device with runtimes like |qnn|_, |tflite|_ and |ort|_.
+|qai_hub|_ is an online platform for developers that simplifies AI model deployment on devices with runtimes like |qnn|_ (QAIRT), |tflite|_ and |ort|_.
 
-Once the AIMET exported model and an encodings JSON file have been obtained, the artifacts can be passed to the |qai_hub| for compilation,
-profiling and inference.
+Once you have obtained an AIMET exported model and a JSON encodings file, you can pass them to the |qai_hub| for compilation, profiling, and inference.
 
-Follow these instructions to `compile AIMET quantized model <https://app.aihub.qualcomm.com/docs/hub/compile_examples.html#compiling-models-quantized-with-aimet-to-tflite-or-qnn>`_ and then submit an inference job using selected device.
+Follow the `instructions <https://app.aihub.qualcomm.com/docs/hub/compile_examples.html#compiling-models-quantized-with-aimet-to-tflite-or-qnn>`_ at the Qualcomm\ |reg| AI hub to compile a model and submit an inference job using the selected device.
```

Comment:
It's not clear to me if this simulates the device or deploys it to your device. What exactly happens here?